creativecloud/blocks/interactive-marquee/interactive-marquee.js

```diff
@@ -74,4 +74,17 @@ function interactiveInit(el) {
 
 export default async function init(el) {
   interactiveInit(el);
+  switch (true) {
+    case el.classList.contains('firefly'): {
+      const buttons = el.querySelectorAll('.con-button');
+      if (buttons.length === 0) return;
```

Comment:
This will not init a firefly marquee at all and return from the init if no button is configured. Is this expected?

Reply:
fixed